cloud-recording/develop/integration-best-practices.md

```diff
@@ -80,14 +81,48 @@ For example, if there is a fixed 5-minute break in each class, you can set `maxI
+
+For scenes with a large audience in the channel or high availability requirements, consider whether you can accept the impact of high availability migration based on your own business characteristics, and decide whether to adopt higher quality assurance measures. For example, create multiple recording tasks for critical scenes using distinct UIDs, and ensure use of servers in different geographical regions through geofencing. Alternatively, you can make periodic API calls and monitor notifications to get the latest recording task status, then create a new cloud recording task with a different UID once you confirm the recording task status is unhealthy.
+
+If you create multiple cloud recording tasks, you are charged separately for each of them. For details, see [Pricing](../reference/pricing).
+
+## Integration requirements checklist
+
+To ensure reliability of the cloud recording service, refer to the following checklist to confirm that your solution meets the integration requirements:
+
+| Serial | Importance | Item | Description |
+|--|----------|----------|----------|
+| 1 | required | Subscribe to a service | Make sure you have activated the cloud recording service. | 
+| 2 | required | request method | <ul><li>To query, use the `POST` request method; to query the recording status use `GET`. </li> <li>Request URLs and request body content are case-sensitive.</li></ul> |
+| 3 | required | Get recording resources | <ul><li>The passed in `uid` cannot duplicate any UID within the current channel.</li><li>For page recording, make sure an appid + cname + uid corresponds to a resource ID.</li><li>Make sure that a resource ID is only used for one cloud recording service.</li><li>Make sure to call the `start` method.</li></ul> |
+| 4 | required | channel scene | Make sure that the channel scene (`channelType`) is consistent with the settings of the <Vg k="VSDK" />. |
+| 5 | required | recording parameters | <ul><li>Make sure that the type, case, and value range of all the parameters passed in when starting the recording are correct, and the required parameters are filled; otherwise, error code 2 is returned.</li><li>Set the layout and video bit rate of the combined recording by referring to the combined layout document and the recording bit rate comparison table.</li></ul> |
+| 6 | required | Confirm that the recording service has started successfully | <ul><li>Make sure that the start request is successful, that is, the `sid` (recording ID) is successfully obtained.</li><li>Make sure to be or within 90 seconds `sid` of getting status 4 5</li></ul> |
```

Comment:
Can this be rephrased to make it clearer? 

> Make sure to be or within 90 seconds `sid` of getting status 4 5

Reply:
Fixed